lib/kitchen/driver/vra.rb

```diff
@@ -0,0 +1,152 @@
+require 'kitchen'
+require 'vra'
+require_relative 'vra_version'
+
+module Kitchen
+  module Driver
+    class Vra < Kitchen::Driver::Base
+      kitchen_driver_api_version 2
+      plugin_version Kitchen::Driver::VRA_VERSION
+
+      required_config :username
+      required_config :password
+      required_config :base_url
+      required_config :tenant
+      required_config :catalog_id
+
+      default_config :subtenant, nil
+      default_config :verify_ssl, true
+      default_config :request_timeout, 600
+      default_config :request_refresh_rate, 2
+      default_config :cpus, 1
+      default_config :memory, 1024
```

Comment:
Not universally, unfortunately... especially in the VMware Hands-On Labs where this will probably get some use.  I'm going to leave it as-is for now and if people complain, I have no problem bumping it.